intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
I was not aware that `bin/rewrite_config_files.py` existed. Do you still think that it should be ordered alphabetically rather than generic and specific?